api/task.go

```diff
@@ -133,7 +133,7 @@ type Task struct {
 	StageID    int `jsonapi:"attr,stageId"`
 	InstanceID int
 	Instance   *Instance `jsonapi:"relation,instance"`
-	// Tasks like creating database may not have database.
+	// The un-done tasks may not have database.
```

Comment:
// Could be empty for creating database task when the task isn't yet completed successfully.